README.md

```diff
@@ -45,21 +46,49 @@ This is the dependency you need:
 </dependency>
 ```
 
-Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
-for more technical details.
+Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto) for more technical details. If you 
+have any question or suggestions, do not hesitate to create as issue, our roadmap is 
+available [here](https://github.com/orgs/artipie/projects/3/views/1).
 
 # Usage
 
-The main types here are:
+The main entities here are:
  - `Storage` interface provides API for key-value storage
  - `Key` represents storage key
  - `Content` represents storage binary value
 
-[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html)
-[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other types are
+[Storage](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Storage.html),
+[Key](https://www.javadoc.io/doc/com.artipie/asto/latest/com/artipie/asto/Key.html) and other entities are
 documented in [javadoc](https://www.javadoc.io/doc/com.artipie/asto/latest/index.html).
 
-TO DO: add more details
+Here is en example of how to create `FileStorage`, save and then read some data:
```

Comment:
`FileStorage` example is well, but there is no description how to use s3 storage.
Maybe we should create a wiki for asto?